frontend/src/app/modules/interventions/list/table/table.component.scss

```diff
@@ -38,18 +38,27 @@
   }
 
   .mat-table .mat-header-cell {
-    border: 10px solid;
-    clip: rect(0 0 0 0);
-    height: 1px;
-    margin: -1px;
-    padding: 0;
-    position: absolute;
-    width: 1px;
+    padding-left: 10px;
+    &:nth-child(1),
```

Comment:
Clever solution and seems to work pretty well, although in the long run also seems to be a bit painful to maintain IMO. 
Let's say we'll do some column reordering, or we'll add a new column in the middle etc - the numbers for nth-child would need to be revised.

What do you think about making it more explicit? E.g adding explicit class "sortable" to the element in the template, and then writing just one selector here:
```
&:not(.sortable) {
 ... same styles as before ...
}
```

This way column reordering or adding/removing columns in the future would have no effect on styling.

Reply:
A little nit: After runing the code, it seems that list items have 5px bottom border separator that looks like a nice shadow and gives some visual separation. 
What do you think about using the same trick to separate the header from the first list item?

Reply:
Maybe even `&:not([mat-sort-header])` selector would suffice to hide all other header, although I am not sure how compiled production build would treat the Angular attributes :)

Reply:
Good ideas! `&:not([mat-sort-header])` works also in production build.
I'd merge this PR and add additional sorting and filtering options in another PR.